my-app/src/Components/Preloader/Preloader.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default function LoadingPopUp() {
+  return (
+    <>
```

Comment:
` <>` empty tags are not needed if you have a common tag that wraps all the markup of the component